src/game/Movement/TargetedMovementGenerator.cpp

```diff
@@ -257,7 +257,7 @@ bool TargetedMovementGeneratorMedium<T, D>::Update(T &owner, const uint32 & time
 
     bool interrupted = false;
     i_recheckDistance.Update(time_diff);
-    if (i_recheckDistance.Passed())
+    if (!i_backing_up && i_recheckDistance.Passed())
```

Comment:
We should preserve the recheck timer for performance reasons rather than bypassing when backing

Reply:
I am not sure what you mean. The "i_backing_up" is a bool member variable, isn't checking that first the most efficient? Also the reason it's there is because the code inside the "if (i_recheckDistance.Passed()))" should not be called on the movespline for backing up, or it will interrupt it since its already in range of the target. Or do you mean you want me to put the "i_backing_up" check in another "if" statement after the "i_recheckDistance.Reset(100)" ?